### ISSUE TITLE
Feature Request: Please add the feature of Bold and Strikthrough rendering of text

### ISSUE THREAD

**Issue by prateekamana** [ttscoff/mdless#78]
![image](https://user-images.githubusercontent.com/41776584/162917725-5de76d97-4da7-4245-b47e-869f3f5f1ffb.png)

**Comment by ttscoff** [comment]
Bold should already get a different color if your theme is set up properly. Not all terminals can support bold fonts, so it doesn’t attempt to do that. I’m not sure how you’d expect strikethrough to render.

Thanks,
Brett

**Comment by prateekamana** [comment]
My md :
```
# TOHAPPEN [12-04-2022]

1. ~~Fix the tooltip bug no matter what. Try not to procrastinate until necessary~~
2. ~~Get FL Studio on Linux working and investigate the prospects of music production on linux~~
3. ❌ Make sense of Search Like Me with Bhavesh's Changes
> Couldn't do because I got tired by the end of the day and crashed early
```

Glow output :
![image](https://user-images.githubusercontent.com/41776584/163143478-b9f47b61-89e2-4557-80a1-bdaf62601af7.png)

mdless output :
![image](https://user-images.githubusercontent.com/41776584/163143653-58e33a9e-9502-4c20-ae1e-404b856ba60a.png)


Please ignore the indentations and whitespace messups, they must just be because of terminal resizing that I did.

**Comment by ttscoff** [comment]
Right, I don't expect any of the strikethrough text to render at this 
point as the syntax isn't recognized by mdless. I'm just trying to 
figure out why the `**` text in your original screenshot isn't 
rendering.

-Brett

**Comment by prateekamana** [comment]
Ah, I'm sorry I forgot my example doesn't have a bold text.
```
# TOHAPPEN [14-04-2022]
1. **Solve the trigger issue ASAP**
2. Make sense of Search Like Me with Bhavesh's Changes
3. Make a finger drumming plan
4. Make a piano learning plan
```
mdless:
![image](https://user-images.githubusercontent.com/41776584/163367920-9491ce08-bffe-4349-8c40-ef7b5b633675.png)

**Comment by ttscoff** [comment]
Huh, apparently it's not parsing bold within a list item. Will have to 
dig into that. It works fine outside in a paragraph or block quote.

-Brett

**Comment by jsclary** [comment]
FYI, it may not be related but bold, italic and bold-italic also have a more general issue in that they seem to require newlline, whitespace or punctuation before the open and after the close so it doesn't properly render legal markdown like
`***x***inu ***i***s ***n***ot ***u***nix`
***x***inu ***i***s ***n***ot ***u***nix

**Comment by ttscoff** [comment]
Because mdless doesn't have a fully tokenized parser, detecting emphasis 
in the middle of a word is problematic because something like 
`file_name` or `any_underscored_word` causes errors, especially when the 
emphasis marker isn't paired. mdless is designed to be a convenience for 
quickly viewing Markdown files, not necessarily a fully fledged Markdown 
engine.

-Brett
